apps/penxle.com/src/routes/(default)/[space]/(setting)/settings/(index)/+page.svelte

```diff
@@ -86,49 +91,49 @@
     name: $query.space.name,
     slug: $query.space.slug,
     description: $query.space.description ?? '',
+    externalLinks: $query.space.externalLinks,
   });
 
-  let links = [
-    { id: 1, url: '첫번째' },
-    { id: 2, url: '두번째' },
-    { id: 3, url: '세번째' },
-  ];
-
-  // let draggedItem: { id: number; url: string } | null = null;
-
-  // const onDragStart = (dataTransfer: DataTransfer | null, link: { id: number; url: string }) => {
-  //   if (dataTransfer) {
-  //     dataTransfer.setData('text/plain', link.url);
-  //     draggedItem = link;
-  //   }
-  // };
-
-  // const onDrop = (link: { id: number; url: string }) => {
-  //   if (!draggedItem) return;
-
-  //   if (draggedItem === link) return;
-
-  //   const indexDragged = links.findIndex((i) => i == draggedItem);
-  //   const indexTarget = links.findIndex((i) => i == link);
-
-  //   if (indexDragged > indexTarget) {
-  //     links = [
-  //       ...links.slice(0, indexTarget),
-  //       draggedItem,
-  //       ...links.slice(indexTarget, indexDragged),
-  //       ...links.slice(indexDragged + 1),
-  //     ];
-  //   } else {
-  //     links = [
-  //       ...links.slice(0, indexDragged),
-  //       ...links.slice(indexDragged + 1, indexTarget + 1),
-  //       draggedItem,
-  //       ...links.slice(indexTarget + 1),
-  //     ];
-  //   }
-
-  //   draggedItem = null;
-  // };
+  const onDragStart = (dataTransfer: DataTransfer | null, link: string) => {
+    if (dataTransfer) {
+      dataTransfer.setData('text/plain', link);
+      draggedItem = link;
+    }
+  };
+
+  const onDrop = (link: string) => {
+    if (!draggedItem) return;
+
+    if (draggedItem === link) return;
+
+    const indexDragged = links.findIndex((i) => i == draggedItem);
+    const indexTarget = links.findIndex((i) => i == link);
```

Comment:
```suggestion
    const indexDragged = links.indexOf(draggedItem);
    const indexTarget = links.indexOf(link);
```

P3: 값 자체를 비교할 경우 predicate function 보단 [`Array.propotype.indexOf`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf) 가 바람직하다고 생각합니다.